lexos/processors/analyze/dendrogrammer.py

```diff
@@ -80,52 +81,54 @@ def get_dendro_distances(linkage_method: str, distance_metric: str,
     :param distance_metric: string, style of distance metric in the dendrogram
     :param linkage_method: string, style of linkage method in the dendrogram
     :param labels: list, file names
-    :return: -silhouette_score: string, containing the result of silhouette
-              score
-             -silhouette_annotation: string, annotation of the silhouette score
-             -score: float, silhouette score
-             -inconsistent_max: float, upper bound of threshold to calculate
-              silhouette score if using Inconsistent criterion
-             -maxclust_max: integer, upper bound of threshold to calculate
-              silhouette score if using Maxclust criterion
-             -distance_max: float, upper bound of threshold to calculate
-              silhouette score if using Distance criterion
-             -distance_min: float, lower bound of threshold to calculate
-              silhouette score if using Distance criterion
-             -monocrit_max: float, upper bound of threshold to calculate
-              silhouette score if using Monocrit criterion
-             -monocrit_min: float, lower bound of threshold to calculate
-              silhouette score if using Monocrit criterion
-             -threshold: float/integer/string, threshold (t) value that users
-              entered, equals to 'N/A' if users leave the field blank
+    :return: - silhouette_score: string, containing the result of silhouette
+               score
+             - silhouette_annotation: string, annotation of the silhouette
+               score
+             - score: float, silhouette score
+             - inconsistent_max: float, upper bound of threshold to calculate
+               silhouette score if using Inconsistent criterion
+             - maxclust_max: integer, upper bound of threshold to calculate
+               silhouette score if using Maxclust criterion
+             - distance_max: float, upper bound of threshold to calculate
+               silhouette score if using Distance criterion
+             - distance_min: float, lower bound of threshold to calculate
+               silhouette score if using Distance criterion
+             - monocrit_max: float, upper bound of threshold to calculate
+               silhouette score if using Monocrit criterion
+             - monocrit_min: float, lower bound of threshold to calculate
+               silhouette score if using Monocrit criterion
+             - threshold: float/integer/string, threshold (t) value that users
+               entered, equals to 'N/A' if users leave the field blank
     """
     # Switch to request.json if necessary
     if request.json:
         opts = request.json
     else:
         opts = request.form
 
-    active_files = len(labels) - 1
+    active_files_num = len(labels) - 1
 
     # since number of labels should be more than 2 and less than n_samples - 1
-    if active_files > 2:
+    if active_files_num > 2:
```

Comment:
content in this if statement is to determine the cluster of dendrogram, create a new function. Maybe `cluster_dendro` or stuff like that, to return `score_label`